functional-test/src/main/java/org/zanata/page/AbstractPage.java

```diff
@@ -484,6 +554,10 @@ public String getHtmlSource(WebElement webElement) {
                 "return arguments[0].innerHTML;", webElement);
     }
 
+    public void triggerScreenshot(String tag) {
+        WebElementUtil.triggerScreenshot(tag);
+    }
+
     // TODO: Deprecated functions, remove after 3.7
```

Comment:
Sure, just thought I'd mention it.

On 3 September 2015 at 09:32, Damian Jansen <notifications@github.com>
wrote:

> In functional-test/src/main/java/org/zanata/page/AbstractPage.java
> <https://github.com/zanata/zanata-server/pull/937#discussion_r38598335>:
>
> > @@ -484,6 +554,10 @@ public String getHtmlSource(WebElement webElement) {
> >                  "return arguments[0].innerHTML;", webElement);
> >      }
> >
> > +    public void triggerScreenshot(String tag) {
> > +        WebElementUtil.triggerScreenshot(tag);
> > +    }
> > +
> >      // TODO: Deprecated functions, remove after 3.7
>
> Maybe in a separate commit?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zanata/zanata-server/pull/937/files#r38598335>.
>



-- 
Sean Flanigan

Principal Software Engineer
Globalisation Tools Engineering
Red Hat